src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -9,12 +10,14 @@
 import com.textmessenger.model.entity.dto.ResponseToFront;
 import com.textmessenger.model.entity.dto.SearchValue;
 import com.textmessenger.model.entity.dto.UserToFrontFull;
-import com.textmessenger.repository.TemporaryTokenRepository;
-import com.textmessenger.service.EmailService;
 import com.textmessenger.service.LoginService;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.textmessenger.repository.TemporaryTokenRepository'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=236&issues=AWYK8i_OvOWp7Yd6HYt3&open=AWYK8i_OvOWp7Yd6HYt3)

[ISSUE_KEY]: AWYK8i_OvOWp7Yd6HYt3